tutorials/dataframe/df103_NanoAODHiggsAnalysis.h

```diff
@@ -0,0 +1,219 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// Header file with functions needed to execute the Python version
+/// of the NanoAOD Higgs tutorial. The header is declared to the
+/// ROOT C++ interpreter prior to the start of the analysis via the
+/// `ROOT.gInterpreter.Declare()` function.
+///
+/// \date June 2019
+/// \author Stefan Wunsch (KIT, CERN), Vincenzo Eduardo Padulano (UniMiB, CERN)
+
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RVec.hxx"
+#include "ROOT/RDF/RInterface.hxx"
+#include "TCanvas.h"
+#include "TH1D.h"
+#include "TLatex.h"
+#include "TLegend.h"
+#include "TLorentzVector.h"
```

Comment:
@stwunsch should we use `ROOT::Math::LorentzVector` instead?

Reply:
Yep!